plugins/inputs/http/http_test.go

```diff
@@ -488,6 +488,9 @@ func TestConnectionOverUnixSocket(t *testing.T) {
 		}
 	}))
 
+	// The Maximum length of the socket path is 104/108 characters, path created with t.TempDir() is too long for some cases
+	// (it combines test name with subtest name and some random numbers in the path)
+	//nolint:usetesting // "os.TempDir() could be replaced by t.TempDir() in TestConnectionOverUnixSocket"
```

Comment:
Same as above, the comment is misleading. Couldn't we just use the comment above the `nolint` statement instead?

Reply:
Wording changed.